addons/initialise-sprite-position/addon.json

```
@@ -1,6 +1,6 @@
 {
   "name": "Customizable new sprite position",
-  "description": "Change the default x/y position of new sprites.",
+  "description": "Change where newly created sprites are placed.",
```

This sounds like, for example, the sprite tile can be placed outside the sprite plane or something like that. I think the current description should be kept since it's more clear.

---

Here's another idea:
```suggestion
  "description": "Change where newly created sprites are placed on the stage.",
```

---

That one works as well; I think it's good.